### PR TITLE
Fixed #19623, missing Pointer class caused errors

### LINKED ISSUE #19623: Many console errors for basic chart following the installation instructions
#### Details
I have followed the most basic of tutorials in the [docs](https://www.highcharts.com/docs/getting-started/installation-with-esm#creating-a-custom-bundle-es6-module) and the console is full of errors already. On page load there is an error and when I mouse over anything I get a stream of other errors (see screenshot). Having ran a debugger I can see that [here](https://github.com/highcharts/highcharts/blob/0043bc371ed03f081616a961453d7d765ec813a4/ts/Core/Series/Series.ts#L4381) the `pointer` variable is being set to `undefined`.

This also happens in Column charts.

#### My code

```
import Chart from 'highcharts/es-modules/Core/Chart/Chart.js';
import LineSeries from 'highcharts/es-modules/Series/Line/LineSeries.js';
// Example to create a simple line chart in a div#container:
const myChart = new Chart('container', {
    series: [
        {
            type: 'line',
            data: [1, 2, 3]
        }
    ]
});
```

I feel like maybe I'm supposed to get a tooltip on hover but as this is my first encounter with the product I'm not sure. But I don't get a tooltip, I just get errors.

#### Expected behaviour
No console errors

#### Actual behaviour
Many console errors
Screenshot 2023-08-25 at 10 41 24

#### Product version
highcharts: 11.1.0

#### Affected browser(s)
Chrome: 115.0.5790.170 MacOs

### PR DESCRIPTION
**Pull request by TorsteinHonsi**
Fixed #19623, errors in custom build when the Pointer class was not added.

Closes #19698.

### CODE REVIEW

File: test.txt
Comment (pawelfus) [inline test.txt:1]: Q: Is this file expected to be here?

**Review by pawelfus — APPROVED** [review]

File: test.txt
Comment (TorsteinHonsi) [reply test.txt:1]: Nope

**Review by bm64 — APPROVED** [review]
Looks good to me!

**Review by bre1470 — APPROVED** [review]
Fantastic! Works for me & looks good. 👍

**Pull request merged**